conjure-core/src/main/java/com/palantir/conjure/defs/SafetyDeclarationRequirements.java

```diff
@@ -18,9 +18,23 @@
 
 public enum SafetyDeclarationRequirements {
     ALLOWED,
-    REQUIRED;
+    REQUIRED,
+
+    EXTERNAL_IMPORTS_REQUIRED;
 
     public boolean required() {
         return this == REQUIRED;
     }
+
+    public boolean externalImportsRequired() {
+        return this == EXTERNAL_IMPORTS_REQUIRED;
+    }
+
+    public boolean allowed() {
+        return this == ALLOWED;
+    }
+
+    public boolean strictMode() {
+        return this == REQUIRED || this == EXTERNAL_IMPORTS_REQUIRED;
+    }
```

Comment:
Thoughts on adding support for safety in a subset of external type imports without changing the validation logic (not in the same commit, anyhow) to keep the change small? If memory serves, we have a custom validation module internally for the project which will benefit most from this, which already limits the allowed external type imports -- it may be better suited to enforce that all valid imports declare safety.

Reply:
That makes sense to me. Would we do the validation here later, or only perform validation in the internal module? Also, could you point me to that internal module?

Reply:
Sent internally :-)

> Would we do the validation here later, or only perform validation in the internal module?

I think this depends on rollout, and whether we intend to support additional types of external type imports later on. For the foreseeable future, it's probably best if we assume only the internal module will handle validation that all external type imports of `java.lang.Long` declare safety.